src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -171,12 +183,15 @@ function check_lifetime(model::AbstractChargeTrappingModel{T}, charge::T, pathti
     return τ
 end
 
-function _constantlifetime_signal!(
-    running_sum::Base.RefValue{T},
-    τ::T,
-    Δt::T,
+function _signal!(
+    ::ConstantLifetimeChargeTrappingModel{T},
     q::Base.RefValue{T},
-    w::T,
+    w::T;
+    running_sum::Base.RefValue{T}=Ref(zero(T)),
+    nσ::T=zero(T),
+    Δl::T=zero(T),
+    τ::T=zero(T),
+    Δt::T=zero(T)
```

Comment:
Here, every keyword argument that is not used can just become `kwargs...` (also in case we might wanna add more charge trapping models in the future).
And maybe do not use `τ = 0` as default value, because that would kill the charge immediately and return `NaN` ^^

```julia
function _signal!(
    ::ConstantLifetimeChargeTrappingModel{T},
    q::Base.RefValue{T},
    w::T;
    running_sum::Base.RefValue{T}=Ref(zero(T)),
    τ::T=T(Inf),
    Δt::T=zero(T),
    kwargs...
) where {T <: SSDFloat}
```